### PR TITLE
Add performance recordings to collections

### LINKED ISSUE #2284: Add collection name lookup during music reserves import
to use data generated from #2266

child of #2043
depends on #2265

### PR DESCRIPTION
**Pull request by hackmastera**
and don't make playlists for them unless they're also in another type of
course

closes #2284

### CODE REVIEW

File: app/services/music_import_service.rb [hunk 1]
```diff
@@ -162,7 +162,7 @@ def ingest_fake_playlist
           # Find previously imported dependent file sets
           file_set_ids = selection_files.map do |file|
             {
-              buffered_change_set_persister.query_service.custom_queries.find_by_property(property: :local_identifier, value: file.entry_id.to_s).first&.id => file.id
+              query_service.custom_queries.find_by_property(property: :local_identifier, value: file.entry_id.to_s).first&.id => file.id
```
Comment (tpendragon) [inline app/services/music_import_service.rb]: I'm a little surprised this works. I guess Sequel holds the transaction open for a given thread?

Comment (hackmastera) [reply app/services/music_import_service.rb]: How would you expect it to break?

Comment (tpendragon) [reply app/services/music_import_service.rb]: It's the query service for the non-buffered adapter, which I would have thought would be outside the transaction, but I guess that's not how it works.

Comment (hackmastera) [reply app/services/music_import_service.rb]: Okay! Yeah I guess it didn't occur to me to think that it might somehow happen outside the transaction.

**Pull request merged**